src/vm/eetwain.cpp

```diff
@@ -5169,7 +5169,11 @@ OBJECTREF EECodeManager::GetInstance( PREGDISPLAY    pContext,
     if (info.ebpFrame)
     {
         _ASSERTE(stackDepth == 0);
+#if defined(_TARGET_X86_) && defined(FEATURE_PAL)
+        taArgBase = GetCallerSp(pContext) - 2 * sizeof(TADDR);
+#else
         taArgBase = *pContext->GetEbpLocation();
```

Comment:
@seanshpark If you intend to use ``pContext->SP`` instead of ``GetCallerSp(pContext)`` in 5173 line, I'm not sure about that. 

As I understand, ``pContext->SP`` stores current stack pointer, and thus it looks strange for me to  infer current frame pointer using ``pContext->SP``.